qutebrowser/mainwindow/tabwidget.py

```diff
@@ -136,18 +136,31 @@ def update_tab_title(self, idx, field=None):
                 (fmt is None or ('{' + field + '}') not in fmt)):
             return
 
+        def right_align(text):
```

Comment:
nit: calling the argument "text" and then using str() to turn it into text because you know it is actually an int is tiny bit misleading. I suggest either calling it something different or assuming the caller passes a string as expected.